RecoHI/HiEgammaAlgos/python/RecoHiEgamma_EventContent_cff.py

```diff
@@ -1,104 +1,79 @@
-    'keep floatedmValueMap_hiRegitMuInitialStepSelector_MVAVals_*', 
-    'keep floatedmValueMap_hiRegitMuMixedTripletStepSelector_MVAVals_*',
-    'keep floatedmValueMap_hiRegitMuPixelLessStepSelector_MVAVals_*', 
-    'keep floatedmValueMap_hiRegitMuPixelPairStepSelector_MVAVals_*', 
-    #'keep recoPFCandidates_*_*_*',
-    'keep recoPFCandidates_particleFlowEGamma_*_*',
-    'keep recoPFCandidates_particleFlowTmp_*_*',
-    "drop recoPFClusters_*_*_*",
-    #"keep recoElectronSeeds_*_*_*",
-    "keep recoElectronSeeds_ecalDrivenElectronSeeds_*_*", 
-    #"keep recoGsfElectrons_*_*_*",
-    "keep recoGsfElectrons_ecalDrivenGsfElectrons_*_*",
-    "keep recoGsfElectrons_electronsWithPresel_*_*",
-    "keep recoGsfElectrons_gedGsfElectronsTmp_*_*",
-    "keep recoGsfElectrons_mvaElectrons_*_*",
-    'keep recoHIPhotonIsolationedmValueMap_photonIsolationHIProducer_*_*',
-    'keep recoPhotons_gedPhotonsTmp_*_*',
-    'keep recoHIPhotonIsolationedmValueMap_photonIsolationHIProducerGED_*_*',
-    'keep recoElectronSeeds_ecalDrivenElectronSeeds_*_*',
-     'keep recoTrackExtras_electronGsfTracks_*_*'
-    )
-    )
+        'keep recoSuperClusters_correctedHybridSuperClusters_*_*',        
+        'keep recoSuperClusters_correctedMulti5x5SuperClustersWithPreshower_*_*',
+        'keep recoSuperClusters_hfEMClusters_*_*',
+        'keep recoSuperClusters_hybridSuperClusters_*_*',
+        'keep recoSuperClusters_multi5x5SuperClusters_*_*',
+        'keep recoSuperClusters_multi5x5SuperClustersWithPreshower_*_*',
+        'keep recoSuperClusters_particleFlowEGamma_*_*',
+        'keep recoSuperClusters_particleFlowSuperClusterECAL_*_*',
+        'keep recoSuperClusters_particleFlowSuperClusterOOTECAL_*_*',
+        'keep recoSuperClusters_lowPtGsfElectronSuperClusters_*_*',
+        'keep recoCaloClusters_hfEMClusters_*_*',
+        'keep recoCaloClusters_particleFlowEGamma_*_*',
+        'keep recoCaloClusters_cleanedHybridSuperClusters_*_*',
+        'keep recoCaloClusters_hybridSuperClusters_*_*',
+        'keep recoCaloClusters_uncleanedHybridSuperClusters_*_*',
+        'keep recoCaloClusters_islandBasicClusters_*_*',
+        'keep recoCaloClusters_multi5x5BasicClustersCleaned_*_*',
+        'keep recoCaloClusters_multi5x5BasicClustersUncleaned_*_*',
+        'keep recoCaloClusters_multi5x5SuperClusters_*_*',
+        'keep recoCaloClusters_particleFlowSuperClusterECAL_*_*',
+        'keep recoCaloClusters_particleFlowSuperClusterOOTECAL_*_*',
+        'keep recoCaloClusters_lowPtGsfElectronSuperClusters_*_*',
+        'keep recoCaloClusters_pfElectronTranslator_*_*',
+        'keep recoCaloClusters_pfPhotonTranslator_*_*',
+        'keep EcalRecHitsSorted_ecalRecHit_*_*',
+        'keep EcalRecHitsSorted_ecalPreshowerRecHit_*_*',
+        'keep EBSrFlagsSorted_ecalDigis__*',
+        'keep EESrFlagsSorted_ecalDigis__*',
+        'keep floatedmValueMap_hiDetachedTripletStepQual_MVAVals_*',        
```

Comment:
As written above, all these floatedmValueMap's are already kept now in the AOD content